### PR TITLE
docs(docs-site): remove prover_endpoints from guide

### PR DESCRIPTION
**Pull request by dionysuzx**

### CODE REVIEW

**Review by davidtaikocha — APPROVED** [review]

File: packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx [hunk 1]
```diff
@@ -18,23 +18,21 @@ These are the bare minimum required settings along with a functional Taiko node
...
-1. A Taiko prover needs to be able to generate SGX proofs at the moment. Please follow the README guide on our [raiko](https://github.com/taikoxyz/raiko/blob/main/docs/README_Docker_and_RA.md) repository.
+1. A Taiko prover needs to be able to generate SGX proofs at the moment. To do that, you need to run raiko. Please follow the Docker setup guide [here](https://github.com/taikoxyz/raiko/blob/main/docs/README_Docker_and_RA.md).
 
-2. Then set the `SGX_RAIKO_HOST` endpoint in your `.env` file.
+2. After successfully setting up raiko, set the `SGX_RAIKO_HOST` endpoint in your `.env` file.
 
 3. Set the `L1_PROVER_PRIVATE_KEY` to an L1 account's private key which will send the Prove Block transactions.
 
-4. Set the `MIN_ACCEPTABLE_PROOF_FEE` to the value in GWei that you deem sufficient.
+4. You must [deploy a `ProverSet` contract](/guides/node-operators/deploy-a-proverset) to run your proposer. If you are working with a third party prover, you will have to follow the guide with them to negotiate allowance and deposit of TAIKO/TTKOh.
```
Comment (YoGhurt111) [inline packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx:27]: Step 4 is optional.
If you keep the proposer and the prover the same EOA, you don't need to deploy a `ProverSet` contract.

File: packages/docs-site/src/content/docs/guides/node-operators/enable-a-proposer.mdx [hunk 2]
```diff
@@ -23,9 +23,9 @@ These are the bare minimum required settings along with a functional Taiko node
 
 <Steps>
 
-1. First, you will need to have a prover from <a href="https://dojonode.xyz/tatami" target="_blank">the community marketplace</a> or [run your own SGX prover](/guides/node-operators/enable-a-prover).
+1. First, you will need to [set up a prover](/guides/node-operators/enable-a-prover). Follow the guide, and note down the contract address for your `ProverSet` contract before continuing here.
 
-2. You MUST set `PROVER_ENDPOINTS` in simple-taiko-node `.env` file to the your SGX prover's endpoint or one from the marketplace. If you do not set it and leave it default while not running a prover, **your proposer will not work**.
+2. Set the contract address in the `PROVER_SET` env variable.
```
Comment (YoGhurt111) [inline packages/docs-site/src/content/docs/guides/node-operators/enable-a-proposer.mdx:28]: Are we just only directing users to use the `ProverSet` way?

File: packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx (hunk 1 above)
Comment (d1onys1us) [reply packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx:27]: but our taiko-client proposer subcommand no longer supports `proverEndpoints` as an arg, right? that is why i believe we decided to tell people to use the latest taiko-client proposer release, you will have to specify your prover via a `proverSetAddress`. so while you can do proposer / prover with the same EOA, there is no way for the latest proposer to specify the prover address unless they use an older version of taiko-client, right?

cc @RogerLamTd @davidtaikocha

File: packages/docs-site/src/content/docs/guides/node-operators/enable-a-proposer.mdx (hunk 2 above)
Comment (d1onys1us) [reply packages/docs-site/src/content/docs/guides/node-operators/enable-a-proposer.mdx:28]: yes, please see comments above. thanks

File: packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx (hunk 1 above)
Comment (YoGhurt111) [reply packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx:27]: Even the latest taiko-client proposer release, we also have 2 ways to set up proposer.
When you keep `proverSetAddress ` empty, you will use your proposer address as your prover address, it also works. You can try it.

Comment (YoGhurt111) [reply packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx:27]: Deploying a contract is a meaningless cost to some users.

File: packages/docs-site/src/content/docs/guides/node-operators/enable-a-proposer.mdx (hunk 2 above)
Comment (d1onys1us) [reply packages/docs-site/src/content/docs/guides/node-operators/enable-a-proposer.mdx]: [5371321](https://github.com/taikoxyz/taiko-mono/pull/17769/commits/537132139fbf1eaa7466b6576180fe82cf4c3961)

File: packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx (hunk 1 above)
Comment (d1onys1us) [reply packages/docs-site/src/content/docs/guides/node-operators/enable-a-prover.mdx]: [5371321](https://github.com/taikoxyz/taiko-mono/pull/17769/commits/537132139fbf1eaa7466b6576180fe82cf4c3961)

**Review by YoGhurt111 — APPROVED** [review]

**Pull request merged**